packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -427,6 +430,17 @@ class SelectableText extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.onSelectionChanged}
   final SelectionChangedCallback? onSelectionChanged;
 
+  /// {@macro flutter.widgets.text_selection.TextMagnifierConfiguration.intro}
+  ///
+  /// {@macro flutter.widgets.magnifier.intro}
+  ///
+  /// {@macro flutter.widgets.text_selection.TextMagnifierConfiguration.details}
+  ///
+  /// By default, builds a [CupertinoTextMagnifier] on iOS and [TextEditingMagnifier] on
```

Comment:
Nit: I put a comment above about the naming of `TextEditingMagnifier`, but maybe it should be called `MaterialTextMagnifier` to follow the `CupertinoTextMagnifier`?